### DISCUSSION TITLE
Token de copiloto inválido: token ausente: 403 /Invalid copilot token: missing token: 403

### DISCUSSION THREAD

**Discussion by paola-rodrigues** [community/community#62415] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Select Topic Area
Bug 
INFO] [auth] [2023-08-01T21:21:06.183Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2023-08-01T21:21:06.189Z] GitHub Copilot could not connect to server. Extension activation failed: "No access to GitHub Copilot found. You are currently logged in as paola-rodrigues."

**Comment by samus-aran** [comment]
Hi @paola-rodrigues 

The most common cause of a 403 error is an attempt to authenticate to Copilot with a GitHub account that does not have a Copilot subscription.

It seems that Copilot is not able to retrieve the correct login information for your work account.

- You should be able to sign out of Copilot (specifically) in Visual Studio by clicking on the Copilot icon (or navigating to Edit -> Copilot Suggestions) and selecting "Sign out".
- If that doesn't work for you, you can close VS, delete the two files (hosts.json and terms.json) in C:\Users%USERNAME%\AppData\Local\github-copilot (or equivalent on a Mac)
- When you restart your editor, you should be prompted to log in.


Let me know how it goes.
Thanks!

**Reply by wangyubow** [reply]
On Ubuntu 22.04, I have the same problem.
Activate VS Code, show the above error message.  Louout Vs Code, and Signin with Github to use Github Copilot, then Copilot is ready to use.

Every time, close Vs Code and start Vs Code, same problem occurs.

Any solution on Ubuntu? 

I failed to find any dir or files like github-copilot or hosts.json or terms.json on Ubuntu 22.04
